packages/adders/drizzle/index.ts

```diff
@@ -66,6 +66,7 @@ const options = defineAdderOptions({
 export default defineAdder({
 	id: 'drizzle',
 	environments: { svelte: false, kit: true },
+	oneOrTwoWordDescription: 'ORM',
```

Comment:
this one's the odd one out given that `i18n` and `css` are all lowercase. `database orm` should be better